build.psm1

```diff
@@ -489,9 +489,15 @@ Fix steps:
     }
 
     # handle TypeGen
-    if ($TypeGen -or -not (Test-Path "$PSScriptRoot/src/System.Management.Automation/CoreCLR/CorePsTypeCatalog.cs")) {
+    $incFileName = 'powershell.inc'
+    if (-not [string]::IsNullOrEmpty($Runtime)) {
+        # File name must be different for Windows and Linux to allow build on Windows and WSL.
+        # Append the first 3 characters fo the Runtime only because otherwise dotnet will crash due to the separation characters. The underscore is only for better readability.
```

Comment:
Thanks for spotting the typo and thanks for coming back about the crash. When I started out I was experimenting with different approaches (one of was to put the inc file into a runtime specific folder) and I actually cannot repro the crash anymore and will therefore use the full `$RunTime` string instead of the first 3 characters.